addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -225,28 +225,15 @@ options.registry.carousel = options.Class.extend({
      * @override
      */
     start: function () {
-        var self = this;
-
+        this.$target.carousel('pause');
         this.$target.carousel({interval: false});
         this.id = this.$target.attr('id');
         this.$inner = this.$target.find('.carousel-inner');
         this.$indicators = this.$target.find('.carousel-indicators');
-        this.$target.carousel('pause');
         this._rebindEvents();
+        this.$target.on('slid.bs.carousel', () => this._retargetEdition());
 
-        var def = this._super.apply(this, arguments);
-
-        // set background and prepare to clean for save
-        this.$target.on('slid.bs.carousel', function () {
-            self.$target.carousel('pause');
```

Comment:
Sure that the carousel does not unpause itself if we control it in any way? Seems odd that this line was there for nothing.

Reply:
The slider widget would start playing again after cloning, because that was triggering a widget refresh on both the original and cloned snippet. Since I have disabled autoplay for the slider widget in edit mode, pausing it once on start should be enough.